projects/begin-wallet-core.md

```diff
@@ -0,0 +1,84 @@
+# Project Name
+Begin Wallet Core
+
+# Preferred Maturity Level
+Growth
```

Comment:
Can you please let us know how you meet the [growth acceptance criteria defined in the project lifecycle](https://tac.openwallet.foundation/governance/project-lifecycle/#growth-stage)?

Reply:
Should be here or in the proposal?

Reply:
We can add it to the proposal.

Reply:
updated